suite-common/suite-config/src/settings.ts

```diff
@@ -1,5 +1,5 @@
 export const settingsCommonConfig = {
     MAX_ACCOUNTS: 10,
     FRESH_ADDRESS_LIMIT: 20,
-    TXS_PER_PAGE: 25,
+    TXS_PER_PAGE: 10,
```

Comment:
We should drop this. For now it is here because the RPC node on free plan cannot handle 25 requests per second.